### ISSUE TITLE
Update variable description on upsert

### ISSUE THREAD

**Issue by codenirvana** [postmanlabs/postman-collection#740]
```javascript
const list = new VariableList(null, [{
    key: 'k1',
    value: 'initial',
    description: 'initial description'
}]);

list.upsert({
    key: 'k1',
    value: 'updated',
    description: 'updated description'
});

list.toJSON();
// [{
//     description: {
//         content: 'initial description',
//         type: 'text/plain'
//     },
//     type: 'any',
//     value: 'updated',
//     key: 'k1'
// }]

```

**Comment by csk1827** [comment]
I could find this bug still reproducible. Can I take up this one?

**Comment by codenirvana** [comment]
@csk1827 Sure 👍

**Comment by codenirvana** [comment]
Fixed by #1332
